streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamGlobalKTableJoinTest.java

```diff
@@ -169,8 +174,10 @@ public void shouldNotJoinOnGlobalTableUpdates() {
         // push all four items to the primary stream. this should produce two items.
 
         pushToStream(4, "X", true, false);
-        processor.checkAndClearProcessResult(new KeyValueTimestamp<>(0, "X0,FKey0+Y0", 2),
-                new KeyValueTimestamp<>(1, "X1,FKey1+Y1", 3));
+        processor.checkAndClearProcessResult(
+            new KeyValueTimestamp<>(0, "X0,FKey0+Y0", 2),
+            new KeyValueTimestamp<>(1, "X1,FKey1+Y1", 3)
+        );
```

Comment:
I had a hard timer reading the test code, so I reformatted it to make it easier to read. Similar elsewhere